security/seccomp-profiles.adoc

```diff
@@ -0,0 +1,77 @@
+[id="seccomp-profiles"]
+= Configuring seccomp profiles
+include::modules/common-attributes.adoc[]
+:context: seccomp-profiles
+
+toc::[]
+
+A container or an OpenShift pod runs a single application that performs one or more well-defined tasks. The application usually requires only a small subset of the underlying operating system kernel APIs.
+Seccomp (secure computing mode) Linux kernel feature can be used to limit the process running in a container to only call a subset of the available system calls. These system calls can be configured by creating profile that is named as seccomp profile.
+
+?? Are below statements important or just a note?
+[IMPORTANT]
+====
+CRI-O runs unconfined by default, without any seccomp profile applied.
```

Comment:
Kubernetes/OpenShift workloads run `unconfined` by default. It's not a limitation by CRI-O.